misc/extension_api_validation/4.2-stable.expected

```diff
@@ -84,6 +84,13 @@ Validate extension JSON: Error: Field 'classes/Sky/properties/sky_material': typ
 Property hints reordered to improve editor usability. The types allowed are still the same as before. No adjustments should be necessary.
 
 
+GH-68420
+--------
+Validate extension JSON: Error: Field 'classes/Node/methods/_get_configuration_warnings/return_value': type changed value in new API, from "PackedStringArray" to "Array".
+
+Allow configuration warnings to refer to a property. Compatibility method registered.
+
+
```

Comment:
This is unrelated to your PR and was reverted in the meantime, please don't re-add it.

Reply:
Good catch, I think at some point it picked it up with an upstream rebase. Should be removed now!